### PR TITLE
GKE - Updated documentations

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
FYI - @toddm92 @mayagoldb 
After the first usage of the integration, we found that the documentation that I release isn't sufficient, Therefor added detailed instructions on how to create and configure GKE API endpoint in the GCP console.

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Review by yaakovi — APPROVED** [review]

File: Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/README.md [hunk 1]
```diff
@@ -12,6 +12,64 @@ Command, argument, and output desctiptions were taken from the Google documentat
 
 * Google Kubernetes Engine Operations Generic Polling.
 
+
+
+## Configure Google Kubernetes Engine setup
+
+1. Select or create a Cloud Platform project:
+
+   <img src="https://github.com/demisto/content/blob/2ee9f543ab14293cef22d46e96f7f29c4b376502/Packs/GoogleKubernetesEngine/doc_files/create-project-1.png" alt="create-project" style="zoom:40%;" />
```
Comment (yaakovi) [inline Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/README.md:21]: change it in all links
```suggestion
   <img src="https://github.com/demisto/content/raw/2ee9f543ab14293cef22d46e96f7f29c4b376502/Packs/GoogleKubernetesEngine/doc_files/create-project-1.png" alt="create-project" style="zoom:40%;" />
```

Comment (GalRabin) [reply Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/README.md]: Replaced all, Thanks

File: Packs/GoogleKubernetesEngine/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "Google Kubernetes Engine",
     "description": "Google Kubernetes Engine managing clusters and node-pools.",
     "support": "xsoar",
-    "currentVersion": "1.0.1",
+    "currentVersion": "1.0.2",
```
Comment (yaakovi) [inline Packs/GoogleKubernetesEngine/pack_metadata.json:5]: don't bump version

**Comment by GalRabin** [comment]
@kirbles19 
I will appreciate your review on:
- [ ] [Readme](https://github.com/demisto/content/blob/a2449cc5a61f77a7272ede19c6ce9b3976b752af/Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/README.md)

**Comment by Itay4** [comment]
@kirbles19 please review

**Comment by GalRabin** [comment]
Status - After meeting with @yaron-libman we go threw the process of creating `credentials`.
@yaron-libman  ETA is 1-3 days for approval.

**Comment by yaron-libman** [comment]
@gal001 reviewed the read me file and ready to go.

**Pull request merged**